### ISSUE TITLE
Mappings - Pages:

### ISSUE THREAD

**Issue by jeremyf** [scientist-softserv/adventist-dl#187]
> Pagination in OAI\_ADL is not mapping to Pages in Hyku. See example: [OAI\_ADL](https://www.google.com/url?q=http://oai.adventistdigitallibrary.org/OAI-script?verb%3DGetRecord%26metadataPrefix%3Doai_adl%26identifier%3D13129455&sa=D&source=editors&ust=1671480230560571&usg=AOvVaw1hyGYPcsZf-2GPaNoBwYji) and [Hyku](https://www.google.com/url?q=https://sdapi.b2.adventistdigitallibrary.org/concern/journal_articles/13129455_13th_sabbath_offering_northern_europe_west_africa?locale%3Den&sa=D&source=editors&ust=1671480230560858&usg=AOvVaw2Kop5yNOaFnoC1tJOH4NsF).

From https://docs.google.com/document/d/1mIOT23UAilSO77pAlXYSWJEHw3YK3BNNQVTKNzd41ao/edit#

**Comment by KatharineV** [comment]
Can confirm this is happening on SDAPI and ADL both. The example above is SDAPI. Here is an example from ADL: [MODS](http://oai.adventistdigitallibrary.org/OAI-script?verb=GetRecord&metadataPrefix=mods&identifier=22252214), [OAI_ADL](http://oai.adventistdigitallibrary.org/OAI-script?verb=GetRecord&metadataPrefix=oai_adl&identifier=22252214), [Hyku](https://adl.b2.adventistdigitallibrary.org/concern/published_works/22252214_prophetic_expositor_and_bible_advocate?locale=en).

**Comment by ShanaLMoore** [comment]
On Hold until client can unblock us. The collection sdapi:article may need to have an ark_id. The import is not successful at its current state. We are unable to view the final imported object, but the raw_metadata and parsed_metadata looks correct. 


see [#collections_created?](https://github.com/samvera-labs/bulkrax/blob/35970cee7b1e48d00c3fc838d2ca5790c09e2c9c/app/models/bulkrax/oai_entry.rb#L51) from Bulkrax. 

[Exhausted timeout](https://sentry.io/organizations/scientist-inc/issues/3827889423/?project=6745020&referrer=project-issue-stream) error in sidekiq.

Jeremy to add this to the client iteration meeting.

**Comment by jeremyf** [comment]
[Seeking feedback in Slack](https://assaydepot.slack.com/archives/C0313NJV9PE/p1674143221910249)

**Comment by jeremyf** [comment]
This is unblocked; our decision is to create collections as we encounter them in the OAI feed.  I amended the testing instructions to now import 30 instead of 1.

**Comment by ShanaLMoore** [comment]
blocked, staging is busted

**Comment by ShanaLMoore** [comment]
block removed, testing on sdapi tenant

**Comment by DiemBTran** [comment]
**This passes SoftServ QA:** See attached screenshots.

tested on:
- importer: https://sdapi.s2.adventistdigitallibrary.org/importers/23?locale=en
- random work's entry on importer: https://sdapi.s2.adventistdigitallibrary.org/importers/23/entries/592?locale=en
- ^that same work's show page: https://sdapi.s2.adventistdigitallibrary.org/concern/journal_articles/13128989__1_million_bible_ministry_makes_major_impact?locale=en

<details><summary>Work show page shows "Pages" metadata</summary>


![Journal Article $1 million Bible ministry makes major impact ID 13128989__1_million_bible_ministry_makes_major_impact Hyku 2023-04-04 at 2 49 12 PM](https://user-images.githubusercontent.com/29311858/229930992-c149f54d-717e-41f8-9303-8fcc34ffd25a.jpg)</details>

**Comment by KatharineV** [comment]
Working as expected on SDAPI Staging! Tested with this [importer](https://sdapi.s2.adventistdigitallibrary.org/importers/27?locale=en).

**Comment by KatharineV** [comment]
Pages metadata is coming through on production as expected for both SDAPI and ADL.
